upload-sourcemaps.js

```diff
@@ -81,6 +81,35 @@ module.exports = async (options) => {
 
     output = uploadResult.stdout.toString();
     log(output);
+
+    setCommits = setCommits || process.env.SENTRY_EXPO_SET_COMMITS;
+    if (setCommits) {
+      let commitsResult = await spawnAsync(
+        sentryCliBinaryPath,
+        ['releases', 'set-commits', '--auto', version],
+        {
+          env: childProcessEnv,
+        }
+      );
+
+      output = commitsResult.stdout.toString();
+      log(output);
+    }
+
+    deployEnv = deployEnv || process.env.SENTRY_EXPO_DEPLOY_ENV;
+    if (deployEnv) {
+      let deployResult = await spawnAsync(
+        sentryCliBinaryPath,
+        ['releases', 'deploys', version, 'new', '-e', deployEnv],
```

Comment:
this will result in an unnamed deploy. Do you think it's useful to add the option to name the deploy - https://docs.sentry.io/api/releases/post-release-deploys/? I haven't used this feature before so I'm not totally sure what the name gives you. You can already use the release version to identify it

Reply:
Ah good point. I don't currently use this but that would be nice to support that too.

Reply:
if there's not a huge reason to add the config option for the deploy name (to try and keep app.json config as slim as we can), we should filter the output with something like
```
output = deployResult.stdout.toString().replace('unnamed ', '');
```